modular_citadel/code/modules/projectiles/guns/pumpenergy.dm

```diff
@@ -158,12 +164,12 @@
 /obj/item/ammo_casing/energy/disabler/slug
 	projectile_type = /obj/item/projectile/beam/disabler/slug
 	select_name  = "overdrive"
-	e_cost = 200
+	e_cost = 300
 	fire_sound = 'modular_citadel/sound/weapons/LaserSlugv3.ogg'
 
 /obj/item/ammo_casing/energy/laser/pump
 	projectile_type = /obj/item/projectile/beam/weak
-	e_cost = 200
+	e_cost = 600
```

Comment:
... Two? _Two?_ Two shots total and that's all it's able to spit out? What kind of shotgun do you think it's supposed to be?

Reply:
To make the numbers clearer, weak lethals deal 15 damage. This round has 3 pellets. 15*3=45. 45*2=90. This nerf makes it literally completely impossible to crit someone with the particle defender's lethal mode, unless you fill your backpack with particle defenders.